R/input-date.R

```diff
@@ -70,14 +74,22 @@
 #'
 #'   # Start with decade view instead of default month view
 #'   dateInput("date6", "Date:",
-#'             startview = "decade")
+#'             startview = "decade"),
+#'
+#'   # Disable Mondays and Tuesdays.
+#'   dateInput("date7", "Date:", daysofweekdisabled = c(1,2)),
+#'   
+#'   # Disable specific dates.
+#'   dateInput("date8", "Date:", value = "2012-02-29",
+#'             datesdisabled = c("2012-03-01", "2012-03-02"))
 #' )
 #'
 #' shinyApp(ui, server = function(input, output) { })
 #' }
 #' @export
 dateInput <- function(inputId, label, value = NULL, min = NULL, max = NULL,
-  format = "yyyy-mm-dd", startview = "month", weekstart = 0, language = "en",
+  format = "yyyy-mm-dd", startview = "month", weekstart = 0,
+  datesdisabled = NULL, daysofweekdisabled = NULL, language = "en",
```

Comment:
Can you move these arguments to the end? Even though this is the right place to put them conceptually, there could be existing code out there that is broken by this change.

Reply:
Sure thing!